src/hacbs/components/Commits/CommitsListView.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+  Button,
+  TextInput,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import { FilterIcon } from '@patternfly/react-icons/dist/js/icons';
+import { Table } from '../../../shared';
+import CommitsListHeader from './CommitsListHeader';
+import CommitsListRow from './CommitsListRow';
+
+const CommitsListView = ({ commits }) => {
+  const [nameFilter, setNameFilter] = React.useState<string>('');
+
+  const filteredComits = React.useMemo(
+    () =>
+      nameFilter ? commits.filter((commit) => commit.sha.indexOf(nameFilter) !== -1) : commits,
+    [nameFilter, commits],
+  );
+
+  const onClearFilters = () => setNameFilter('');
+  const onNameInput = (name: string) => setNameFilter(name);
+  return (
+    <>
+      <Toolbar data-testid="component-list-toolbar" clearAllFilters={onClearFilters}>
+        <ToolbarContent>
+          <ToolbarGroup alignment={{ default: 'alignLeft' }}>
+            <ToolbarItem>
+              <Button variant="control">
+                <FilterIcon /> {'Name'}
+              </Button>
+            </ToolbarItem>
+            <ToolbarItem>
+              <TextInput
+                name="nameInput"
+                data-testid="name-input-filter"
+                type="search"
+                aria-label="name filter"
+                placeholder="Filter by name..."
+                onChange={(name) => onNameInput(name)}
+              />
+            </ToolbarItem>
+          </ToolbarGroup>
+        </ToolbarContent>
+      </Toolbar>
+      <Table
+        data={filteredComits}
```

Comment:
Filtered empty state contains default `not found` text

![image](https://user-images.githubusercontent.com/9964343/185400264-2835b82c-1def-4aaa-94c6-acc1241fc7bf.png)


When the filtered results are empty, we show an empty state in other places.
here is a screenshot from Christian's Environment PR:  `Settings -> Environment`.  we can follow this as a convention in all the list view with filters. cc:  @christianvogt  @Ranelim

![image](https://user-images.githubusercontent.com/9964343/185399014-5d451378-849c-41c7-94cd-8468eede7405.png)